infra/src/dsl/types/output-types.ts

```diff
@@ -4,6 +4,7 @@ import { ReferenceResolver, EnvironmentConfig } from './charts'
 // Output types
 export type ContainerRunHelm = {
   command: string[]
+  image?: string
```

Comment:
Seems to be missing as an input into the helm chart.

Reply:
I am not following

Reply:
@peturgq bump

Reply:
Here you're only setting the initcontainer image to be the same as the service image https://github.com/island-is/island.is/blob/758a34831f77b57df9813cf760a8481266fa3529/infra/helm/libs/api-template/templates/deployment.yaml#L72

So we need something in the template that reads and sets the initcontainer image from the init container value.

Reply:
Ahh!

Well spotted, that I didn't see :pray:

Reply:
let's keep it in scope, see updated template